src/server_manager/tsconfig.json

```diff
@@ -10,5 +10,8 @@
     // FIXME: these tests fail with a runtime error because app.ts depends on
     // polymer, which targets the browser and uses ES6 imports.
     "web_app/app.spec.ts"
-  ]
+  ],
+  "compilerOptions": {
```

Comment:
Is this really needed? How come this was working before? I thought webpack would take care of those things.
/cc @alalamav

Reply:
What do you need from ES2020?

You shouldn't have to specify multiple ES versions.

Reply:
It pulls in [This tsc bugfix](https://github.com/microsoft/TypeScript/pull/38013#issuecomment-646314376) which lives, as per this comment (and also I confirmed by checking) in `es2020.intl.d.ts`.  This is how we pull in that version of `Intl`.

Reply:
Let's install the Format.JS polyfill instead: https://formatjs.io/docs/polyfills/intl-numberformat

This way we don't need to make any changes to the existing build. You will get the right types, without having to change tsconfig or the TS version.

Reply:
Using the polyfill requires dynamically importing a new module for each language change.  This runs into an error: "Dynamic imports are only supported when the '--module' flag is set to 'es2020', 'esnext', 'commonjs', 'amd', 'system', or 'umd'". So we will have to set these flags anyways to use the polyfill.  Because of this I chose to use the system supported version since this is only used in browser code, not in Node.

Reply:
If we are assuming the code is running on an ES2020 environment, then set the target to ES2020 and revert the lib option. Otherwise it's very confusing.

Reply:
After extensive offline work, we decided to get rid of the compiler options and cast away the type issues.  I added a big warning at the top of the file that we're using an implicit ES2020 dependency for browser code here.